pkg/standalone-utils/contracts/PauseHelper.sol

```diff
@@ -0,0 +1,144 @@
+     * @param pool Pool that was removed
+     */
+    event PoolRemovedFromPausableSet(address pool);
+
+    EnumerableSet.AddressSet private _poolSet;
+
+    constructor(IVault vault) SingletonAuthentication(vault) {
+        // solhint-disable-previous-line no-empty-blocks
+    }
+
+    /***************************************************************************
+                                                      Manage Pools
+    ***************************************************************************/
+
+    /**
+     * @notice Add pools to the list of pools that can be paused.
+     * @dev This is a permissioned function. Only authorized accounts (e.g., monitoring service providers) may add
+     * pools to the pause list.
+     * @param newPools List of pools to add
+     */
+    function addPools(address[] calldata newPools) external authenticate {
+        uint256 length = newPools.length;
+
+        for (uint256 i = 0; i < length; i++) {
+            if (_poolSet.add(newPools[i]) == false) {
+                revert PoolExistInPausableSet(newPools[i]);
+            }
+
+            emit PoolAddedToPausableSet(newPools[i]);
+        }
+    }
+
+    /**
+     * @notice Remove pools from the list of pools that can be paused.
+     * @dev This is a permissioned function. Only authorized accounts (e.g., monitoring service providers) may remove
+     * pools from the pause list.
+     * @param pools List of pools to remove
+     */
+    function removePools(address[] memory pools) public authenticate {
+        uint256 length = pools.length;
+        for (uint256 i = 0; i < length; i++) {
+            if (_poolSet.remove(pools[i]) == false) {
+                revert PoolNotFoundInPausableSet(pools[i]);
+            }
+
+            emit PoolRemovedFromPausableSet(pools[i]);
+        }
+    }
+
+    /**
+     * @notice Pause a set of pools.
+     * @dev This is a permissioned function. Governance must first grant this contract permission to call `pausePool`
+     * on the Vault, then grant another account permission to call `pausePools` here. Note that this is not necessarily
+     * the same account that can add or remove pools from the pausable list.
+     *
+     * Note that there is no `unpause`. This is a helper contract designed to react quickly to emergencies. Unpausing
+     * is a more deliberate action that should be performed by accounts approved by governance for this purpose, or by
+     * the individual pools' pause managers.
+     * @param pools List of pools to pause
+     */
+    function pausePools(address[] memory pools) public authenticate {
+        uint256 length = pools.length;
+        for (uint256 i = 0; i < length; i++) {
+            if (_poolSet.contains(pools[i]) == false) {
+                revert PoolNotFoundInPausableSet(pools[i]);
+            }
+
+            getVault().pausePool(pools[i]);
+        }
+    }
+
+    // --------------------------  Getters --------------------------
```

Comment:
Use the standard block header style if you want one here:

    /***************************************************************************
                                   Getters
    ***************************************************************************/

With a blank space before the content.